### PR TITLE
Repart test

### PR DESCRIPTION
**Pull request by xnox**
testing ubuntu ci

### CODE REVIEW

**Comment by xnox** [comment]
Hopefully, Ubuntu CI will now install any missing files into systemd package. Such that repart PR can test correctly.

**Comment by xnox** [comment]
Well it did install repart

```
make[1]: Entering directory '/tmp/autopkgtest.ai4NKf/build.Jsi/systemd'
dh_missing --sourcedir debian/install/deb --list-missing 2>&1 | sed -n 's|dh_missing: \(.*\) exists in debian/install/deb but is not installed to anywhere|\1|p' | xargs --verbose -L1 dh_install --sourcedir debian/install/deb -psystemd
dh_install --sourcedir debian/install/deb -psystemd bin/systemd-repart 
make[1]: Leaving directory '/tmp/autopkgtest.ai4NKf/build.Jsi/systemd'
```

but it also installed like all of udev sutff into systemd package too.

**Comment by xnox** [comment]
Ok try again.

File: man/partition.d.xml [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+  <refsect1>
+    <title>Description</title>
+
+    <para><filename>partition.d/*.conf</filename> files describe basic properties of partitions of block
+    devices of the local system. They may be used to declare types, names and sizes of partitions that shall
+    exist. The
+    <citerefentry><refentrytitle>systemd-repart</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+    service reads these files and attempts to add new partitions currently missing and enlarge existing
+    partitions according to these definitions. Operation is generally incremental, i.e. when applied, what
+    exists already is left intact, and partitions never shrunk, moved or deleted.</para>
```
Comment (behrmann) [inline man/partition.d.xml:37]:
```suggestion
    exists already is left intact, and partitions are never shrunk, moved or deleted.</para>
```

File: man/systemd-repart.xml [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><command>systemd-repart</command> grows and adds partitions to a partition table, based on the
+    configuration files described in
+    <citerefentry><refentrytitle>partition.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
+    </para>
+
+    <para>If invoked with no arguments, operates on the block device backing the root file system partition
```
Comment (behrmann) [inline man/systemd-repart.xml:43]:
```suggestion
    <para>If invoked with no arguments, it operates on the block device backing the root file system partition
```
I think there is an "it" missing here

**Comment by xnox** [comment]
@mrc0mmand @poettering if you retrigger repart PR it should work correctly now w.r.t. ubuntu testing. Look it is all green here.

@behrmann thanks for comments, but this is a PR to test that my CI changes are correct. The main repart PR is at https://github.com/systemd/systemd/pull/14368 I'm not sure how to transfer your suggestions there.

**Comment by behrmann** [comment]
@xnox sorry for posting in the wrong PR, I usually only skim the most recent PRs to see what's been going on recently and did wonder why the PR was from you, but the commits weren't. I added the suggestions to #14368, thanks for the heads up!

**Pull request closed without merging**